Core/CwCheat.cpp

```diff
@@ -440,6 +440,10 @@ struct CheatOperation {
 	uint32_t addr;
 	int sz;
 	uint32_t val;
+	uint16_t vibrL;
```

Comment:
To save space, it'd be better to put these as a struct inside the union.  All ops use this same structure, so right now this is wasting probably 8 bytes per cheat.  It's not that big, but still, the other types are all using the unioned structs, best to stay consistent.

-[Unknown]